core/server/data/timezones.json

```diff
@@ -2,328 +2,263 @@
 	"timezones": [
 		{
 			"name": "Pacific/Pago_Pago",
-			"label": "(GMT -11:00) Midway Island, Samoa",
-			"offset": -660
+			"label": "(GMT -11:00) Midway Island, Samoa"
```

Comment:
from what i understood i thought we also remove the offset in the label as well.
if not, i'm fine 👍